### ISSUE TITLE
question about ade20k benchmark learning setting

### ISSUE THREAD

**Issue by Sunting78** [facebookresearch/MaskFormer#41]
A great work! Here, I have some questions.
1. val data is used to fine-tune the model by training data or train and val data learn together?
2. if val to fine-tune, how much epoch is used? Is there any unify and default setting for fair comparison?
3. if train and val data learn together, the learning setting is also the same? 160k iters?

Look forward to your reply.

**Comment by Sunting78** [comment]
> Sorry, I did not get your question. All numbers reported in our paper are trained on `train` set and evaluated on `val` set.

I want to ask the **'test' set** result on ade20k.  The rank 1st in ADE20K leaderboard.  How to use val to get test set score? The detail learning method is not mentioned in paper.

**Comment by bowenc0221** [comment]
I have updated the arxiv paper with test set results in the appendix. Please check it for more detail. We train our model on train + val set only once with the same number of epochs, and we did not tune any parameters for test set numbers.
